reports/sks444.md

```diff
@@ -0,0 +1,246 @@
+ - 14:
+   - repo: community
+   - link: https://github.com/coala/community/commit/d8e760bf8abd555ef643652bdcdf652fda5575ee
+   - details: >
+     data/tests/: Add tests for data app
+ - 15:
+   - repo: community
+   - link: https://github.com/coala/community/commit/baae22188f1880fd6fd4102a4a3f12ecce87381a
+   - details: >
+     test_management_commands.py: Fix webservices error
+ - 16:
+   - repo: community
+   - link: https://github.com/coala/community/commit/e09ae10579481ac61b28d59419c4eddc4c78f83c
+   - details: >
+     data/: Fix import_contributors_data command
+ - 17:
+   - repo: community
+   - link: https://github.com/coala/community/commit/829ceba7f4ddf829c531a18b0413b6d0e883af0b
+   - details: >
+     test_models.py: Fix tests
+ - 18:
+   - repo: community
+   - link: https://github.com/coala/community/commit/34748e4b9bcb39bb659edba1bda458cc32734049
+   - detials: >
+     Get 100% test coverage for the data app
+ - 19:
+   - repo: community
+   - link: https://github.com/coala/community/commit/537602ecbe67263a1c3a877f88444ec6620190f4
+   - details: >
+     data/: Import IGitt data
+ - 20:
+   - repo: community
+   - link: https://github.com/coala/community/commit/68556ccb73df543f3e6b75092c787243611fda94
+   - details: >
+     data/: Add url field to Issue and MergeRequest model
+ - 21:
+   - repo: community
+   - link: https://github.com/coala/community/commit/ef814f6e774a9d04da5a28ee35e2b3a55229694b
+   - details: >
+     Add gamification app
+ - 22:
+   - repo: community
+   - link: https://github.com/coala/community/pull/153/commits/7bca77cb7261d391132c895c7cdf1b8fba556785
+   - details: >
+     [WIP] openhub/tests/: Create tests for openhub app
+ - 23:
+   - repo: community
+   - link: https://github.com/coala/community/pull/145/commits/b1ca875cac0aae686d9da454621b07354135f469
+   - details: >
+     [WIP] gitter/: Messages, Analysis and Import
+ - 24:
+   - repo: community
+   - link: https://github.com/coala/community/pull/152/commits/3c7173128f884d3b618861355f41f4a2836ab252
+   - details: >
+     [WIP] openhub/data.py: Convert data dict to list
+ - 25:
+   - repo: igitt-django
+   - link: https://gitlab.com/sks444/igitt-django/commit/824ced17fc6fbff379aef850ceccb5779d1f5a3e
+   - details: >
+     Switch to sqlite3
+ - 26:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/45/diffs?commit_id=993a68252f01c4ae7d78bfec22ba972b2fc5906e
+   - details: >
+     [Not Reviewed] Serialize messages on gitter by newcomers
+ - 27:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/59/diffs?commit_id=0581b4c81c981eca81541b7f312beabf97393537
+   - details: >
+     [Not Reviewed ] org/: Use igitt_django models to save igitt data
+ - 28:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/60/diffs?commit_id=25e750d6f4f26e3dc7a10fabd8897a23e27d4332
+   - details: >
+     [Not Reviewed] org/: Make an api to show active newcomers
+ - 29:
```

Comment:
29 here and 27 in the tarball ... what happened ?

Reply:
@jayvdb updated the tarball, it contains 28 patches now, it doesn't contain https://github.com/sks444/webservices.coala.io/commit/9d263725 as there is nocode, it just has json.